tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1874,6 +1874,7 @@ def resize_image_with_pad_v1(image,
 
   Raises:
     ValueError: if `target_height` or `target_width` are zero or negative.
+    InvalidArgumentError: if certain positive value<100, is given for target_width.
```

Comment:
```suggestion
    InvalidArgumentError: if `target_width < 100`
```

Please also make sure that this is the right threshold, if we go via this route.